src/test/java/gil/server/HTTPParserTest.java

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
For this class I see more methods than tests, which shows that you're not following TDD. It is very important that you TDD and write a test _before_ you write implementation. Even if your methods are covered in the acceptance tests, it's important (in general and for your review board, as well) that they also be covered at the unit level.

Reply:
Good point. I just refactored it.